vignettes/p2visualisation.Rmd.orig

```diff
@@ -0,0 +1,217 @@
+---
```

Comment:
I am not sure which is the correct Rmd file. Please see comments above in case.

Reply:
`.orig` is the original and one that is updated. `precompile.R` precompiles all these vignettes, which helps us get around the vignettes taking too long to run for CRAN.